civic_api_client/variants_lister.py

```diff
@@ -377,12 +377,27 @@ def get_variant_details(self, variant_id):
         variant_url = utils.civic_api_url() + 'variants/' + str(variant_id)
         return requests.get(variant_url, verify = False).json()
 
+    def evi_valid(self,variant_detail):
+        "Fliter variant with no accepted evidence item"
+        valid = False
+        if 'evidence_items' in variant_detail:
+            evi_items = variant_detail['evidence_items']
+            if len(evi_items) == 0:
+                return valid
+            else:
+                for evi in evi_items:
+                    if evi['status'] ==  "accepted":
+                        valid = True
+        return valid
+
+
```

Comment:
This could also be written as so,
```python
+    def evi_valid(self,variant_detail):
 +        "Fliter variant with no accepted evidence item"
 +        valid = False
 +        if 'evidence_items' in variant_detail:
 +            for evi in variant_detail['evidence_items']:
 +                if evi['status'] ==  "accepted":
 +                    valid = True
 +        return valid